elixir/lib/homework/transactions/transaction.ex

```diff
@@ -20,7 +20,15 @@ defmodule Homework.Transactions.Transaction do
   @doc false
   def changeset(transaction, attrs) do
     transaction
-    |> cast(attrs, [:user_id, :amount, :debit, :description, :merchant_id])
-    |> validate_required([:user_id, :amount, :debit, :description, :merchant_id])
+    |> cast(attrs, [:user_id, :amount, :credit, :debit, :description, :merchant_id])
+    |> validate_required([:user_id, :amount, :credit, :debit, :description, :merchant_id])
+    |> validate_number(:amount, greater_than: 0)
+    |> check_constraint(
```

Comment:
I'm not sure if this is required, but I'm assuming that a transaction can only be a debit or a credit, but not both.  

I'm pretty sure this is referring to "debit card" or "credit card", but even if it were Accounting Debits and Credits, the same logic would apply.